build.gradle

```diff
@@ -8,8 +8,8 @@ import java.nio.file.Paths
 
 buildscript {
     ext {
-        opensearch_version = System.getProperty("opensearch.version", "3.0.0-beta1-SNAPSHOT")
-        buildVersionQualifier = System.getProperty("build.version_qualifier", "beta1")
+        opensearch_version = System.getProperty("opensearch.version", "3.0.0-SNAPSHOT")
+        buildVersionQualifier = System.getProperty("build.version_qualifier", "")
```

Comment:
Can remove this since we have version bump PR already https://github.com/opensearch-project/flow-framework/pull/1124